web/src/composables/useLogs.ts

```
@@ -4342,6 +4412,721 @@
+        position: "bottom",
+        timeout: 4000,
+      });
+    }
+  };
+
+  const handlePageCountError = (err: any) => {
+    searchObj.loading = false;
+    let trace_id = "";
+    searchObj.data.countErrorMsg = "Error while retrieving total events: ";
+    if (err.response != undefined) {
+      if (err.response.data.hasOwnProperty("trace_id")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)